src/SideBar/SideBar.jsx

```diff
@@ -0,0 +1,21 @@
+import data from "../Data/stretch-goal.json"
+
+export const SideBar = ()=>{

```

Comment:
Nice to see that you implemented this stretch goal! I would strongly suggest you have another look at how cramped it gets on mobile though. The space is scarce, and the elements become very small

Reply:
I improve the style for the small screen. Surly, it's not as perfect as I would wish for. I will explore and work more on it when I get time later! :)